releasenotes/notes/1.1/add-elide-permutations-to-pipeline-077dad03bd55ab9c.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-features:
+features_transpiler:
   - |
     The transpiler pass :class:`~.ElidePermutations`
     runs by default with optimization level 2 and 3. Intuitively, removing
```

Comment:
I'd suggest rewording the end of this release note a bit (I can't leave a suggestion because it's too far from the diff), to something like:

```
As :class:`~.OptimizeSwapBeforeMeasure` is a special case
of :class:`~.ElidePermutations`, it has been replaced by the
:class:`.ElidePermuations` pass as part of the init stage in
optimization level 3 pass managers.